package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "build": "turbo run build",
-    "checks": "turbo run checks",
+    "checks": "turbo run build && turbo run checks",
```

Comment:
what is it fixing? build is kinda expensive, you don't want to have to run it every time you need checks

Reply:
that's the problem we need to fix differently I think cc @andrasbacsai

Reply:
please remove the build here, we will soon have a solution for typescript (probably this week)

Reply:
ping